dapr/proto/sentry/v1/README.md

```diff
@@ -0,0 +1,3 @@
+# Sentry Service APIs
+
+This folder is intended for Sentry Service APIs that the Daprd Sidecars use to manage mTLS between services and acts as a certificate authority. [See more on the Sentry Control Plane Service in the Dapr Docs](https://docs.dapr.io/concepts/dapr-services/sentry/).
```

Comment:
Sentry is used by the Dapr control plane services too

Reply:
```suggestion
This folder is intended for `sentry` service APIs that the `daprd` sidecars use to manage mTLS between services and acts as a certificate authority. More details about the `sentry` please refer to the [docs](https://docs.dapr.io/concepts/dapr-services/sentry/).
```